lib/routes/api/accounts.js

```diff
@@ -176,6 +176,7 @@ function validateUpdateCall(opts) {
     'child_call_hook',
     'call_status',
     'listen_status',
+    'transcribe_status',
```

Comment:
this is the only change?

Reply:
yes this is the only one change, When LCC happens, api-server check if the LCC body contains expected command, currently api-server is not accept LCC body has transcribe_status command